Project21/Project21/intlist.c

```diff
@@ -0,0 +1,212 @@
+// ------------------------------------------------------------------------------------------------
+// Assignment 2 ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------
+// intlist.c
+// c file for creating a linked list
+// ------------------------------------------------------------------------------------------------
+#include"intlist.h"
+#include<stdlib.h>
+#include<stdio.h>
+#include<stdbool.h>
+#include <malloc.h>
+
+bool deleted;// Boolean varaible to keep track of deleted list 

```

Comment:
If you create more than one list, delete the first and the check whether the both lists are deleted, this would return true for both.